src/components/PublicHeader.tsx

```diff
@@ -23,6 +23,13 @@ const PublicHeader = ({ user, org, userMemberships } : PublicHeaderProps) : JSX.
         membership => membership.role) || [];
 
     const canOrganize = userMemberships.length > 0;
+    let currentOrg = '';
+
+    for (let i = 0 ; i < userMemberships.length ; i++) {
+        if (userMemberships[i].organization.id == org?.id && userMemberships[i].role == 'admin') {
+            currentOrg = org?.id.toString();
+        }
+    }
```

Comment:
Can this be expressed in a more concise way? 
We are already filtering the user memberships on line 22 to check if they are memberships in which the user has any kind of role (user doesn't need to be an admin to organize, just needs to have any role). So on line 29 it is redundant to check for an admin role again. 

One suggestion is to use an array method instead of a for loop, for less lines of code -- maybe something like 
`const currentOrg = userMemberships.find(membership => membership.organization.id == org.id)`

this will return the first membership that satisfies the testing condition, or undefined if nothing does.

and then assign a variable for the organize url that you can refer to when rendering -- 

`const organizeUrl = currentOrg? `/organize/${currentOrg.organization.id}` : `/organize``

There wouldn't be any need to convert the number to a string explicitly with .toString() inside the template literal